procrastinate/contrib/django/admin.py

```diff
@@ -106,12 +119,19 @@ def short_task_name(self, instance: models.ProcrastinateJob) -> str:
 
     @admin.display(description="Args")
     def pretty_args(self, instance: models.ProcrastinateJob) -> str:
-        indent = 2 if len(instance.args) > 1 or len(str(instance.args)) > 30 else None
-        pretty_json = json.dumps(instance.args, indent=indent)
-        if len(pretty_json) > 2000:
-            pretty_json = pretty_json[:2000] + "..."
+        arg_rows = [
+            format_html(
+                "<tr><td>{key}</td><td>{value}</td></tr>",
+                key=key,
+                value=ellipsize_middle(value),
```

Comment:
I think we should `json.dump()` the value though. Given we know it's json, I guess the only difference will be boolean capitalization and null/None. But here in the django admin, it's not about being Python anymore, and json types are more "neutral".

Reply:
(Also, even if we don't `json.dump()`, your IDE should probably complain: value has no reason to be a string, it might be any json type. I guess PyRight should complain in the CI, if it doesn't I'll be somewhat disappointed at another Microsoft product :D (said the guy on GitHub, using VSCode )

Reply:
(No idea how it even worked before, as shown in your screenshot...)